tmpl/wiki/hello-world.md

```diff
@@ -142,131 +154,287 @@ If you are on a 64-bit Linux system able to build Xen images, simply change
 `--unix` for `--xen` to build a Xen VM:
 
 ```
-mirage configure --xen
-make depend
-make
+$ mirage configure --xen
+$ make
 ```
 
-*Everything* else remains the same!  The `main.ml` and `Makefile` generated
+*Everything* else remains the same! The `main.ml` and `Makefile` generated
 differ significantly, but since the source code of your application was
-parameterised over the `CONSOLE` type, it doesn't need to be changed to run
-using the Xen console driver instead of Unix.
+parameterised over the `CONSOLE` type, it doesn't matter-- you do not need to
+make any changes for your code to run when linked against the Xen console driver
+instead of Unix.
 
 When you build the Xen version, you'll have a `mir-console.xen` unikernel that
 can be booted as a standalone kernel. You will also have a `console.xl` Xen
 configuration file in the current directory that looks something like this:
 
 ```
-# Generated by Mirage (Tue, 31 Dec 2013 19:27:12 GMT).
+# Generated by Mirage (Tue, 14 Jul 2015 11:17:40 GMT).
 
 name = 'console'
-kernel = '/home/avsm/src/git/avsm/mirage-www/src/mir-console.xen'
+kernel = '/home/vagrant/mirage-skeleton/console/mir-console.xen'
 builder = 'linux'
 memory = 256
-```
-
-Edit this to customize the VM name or memory, and then run it via `xl create -c console.xl`
-(or, if you're still on the older Xen, swap the `xl` command for `xm`).
-You should see the same output on the Xen console as you did on the
-UNIX version you ran earlier. If you need more help, or would like to boot your
-Xen VM on Amazon's EC2, [click here](/wiki/xen-boot).
-
-If you are using an ARM processor, such as the Cubie Board, you will need some extra pins that can be found [here](http://openmirage.org/blog/introducing-xen-minios-arm).
+on_crash = 'preserve'
+
+# You must define the network and block interfaces manually.
+
+# The disk configuration is defined here:
+# http://xenbits.xen.org/docs/4.3-testing/misc/xl-disk-configuration.txt
+# An example would look like:
+# disk = [ '/dev/loop0,,xvda' ]
+
+# The network configuration is defined here:
+# http://xenbits.xen.org/docs/4.3-testing/misc/xl-network-configuration.html
+# An example would look like:
+# vif = [ 'mac=c0:ff:ee:c0:ff:ee,bridge=br0' ]
+```
+
+Edit this to customise the VM name or memory, and then run it (if you're still
+on an old release of Xen you may need to swap the `xl` command for `xm`):
```

Comment:
xl became default in xen-4.2 and xm was removed in xen-4.5. I would suggest: use xm if xend is running and xl otherwise.